src/c++/perf_analyzer/genai-pa/genai_pa/main.py

```diff
@@ -40,6 +41,7 @@
 # Optional argv used for testing - will default to sys.argv if None.
 def run(argv=None):
     args = parser.parse_args(argv)
+    utils.remove_file(args.profile_export_file)
```

Comment:
Nice cleanup since last review! I would move this `remove_file` call into the `wrapper.py` logic just before calling `perf_analyzer`. This `main.py` code is agnostic to whatever the wrapper/args are doing.

Reply:
yup. thats a good call. moved